### PR TITLE
move to version 0.0.58

### PR DESCRIPTION
**Pull request by johnsteele**
Signed-off-by: John Steele <programjsteele@gmail.com>

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by benoitf** [comment]
@johnsteele could you please update as well https://github.com/eclipse/che-plugin-registry/blob/master/che-theia-plugins.yaml#L279-L304 
(we'll soon have only this file)

File: v3/plugins/redhat/mta-vscode-extension/0.0.58/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
```
Comment (ericwill) [inline v3/plugins/redhat/mta-vscode-extension/0.0.58/meta.yaml:1]: This change can be removed -- `meta.yaml` files are generated via the `che-theia-plugins.yaml` file now.

File: v3/plugins/redhat/mta-vscode-extension/latest.txt [hunk 2]
```diff
@@ -1 +1 @@
-0.0.56
```
Comment (ericwill) [inline v3/plugins/redhat/mta-vscode-extension/latest.txt:1]: This change can also be removed.

File: vscode-extensions.json [hunk 3]
```diff
@@ -396,7 +396,7 @@
     },
     {
       "repository": "https://github.com/windup/rhamt-vscode-extension",
-      "revision": "ece569b2a6222ee6b2a16ede8241e4817a574ea4"
+      "revision": "5ae3dd17a290b249fecea4a031b8debd71b5da24"
```
Comment (ericwill) [inline vscode-extensions.json:399]: This change can also be removed, this file doesn't exist anymore (it's been replaced by `che-theia-plugins.yaml`)

**Review by ericwill — CHANGES_REQUESTED** [review]
Please rebase and apply changes

**Comment by ericwill** [comment]
Obsolete PR

**Pull request closed without merging**